src/__tests__/unit/lib/boavizta/index.test.ts

```diff
@@ -64,6 +64,35 @@ describe('lib/boavizta: ', () => {
         ]);
       });
 
+      it('returns a result when the impacts from the API is undefined.', async () => {
```

Comment:
why we are returning valid result if the API response is undefined?

Reply:
updated description. We set it to 0 in case of there isn't data from the API